editor/editor_about.cpp

```diff
@@ -105,11 +126,46 @@ ScrollContainer *EditorAbout::_populate_list(const String &p_name, const List<St
 			il->set_same_column_width(true);
 			il->set_auto_height(true);
 			il->set_mouse_filter(Control::MOUSE_FILTER_IGNORE);
+			il->set_focus_mode(Control::FOCUS_NONE);
 			il->add_theme_constant_override("h_separation", 16 * EDSCALE);
-			while (*names_ptr) {
-				il->add_item(String::utf8(*names_ptr++), nullptr, false);
+			if (p_allow_website) {
+				il->set_focus_mode(Control::FOCUS_CLICK);
+				il->set_mouse_filter(Control::MOUSE_FILTER_PASS);
+
+				il->connect("item_activated", callable_mp(this, &EditorAbout::_item_with_website_selected).bind(il));
+				il->connect("resized", callable_mp(this, &EditorAbout::_item_list_resized).bind(il));
+				il->connect("focus_exited", callable_mp(il, &ItemList::deselect_all));
+
+				while (*names_ptr) {
+					const String name = String::utf8(*names_ptr++);
+					const String identifier = name.get_slice(" <", 0);
+					const String website = name.get_slice_count(" <") == 1 ? "" : name.get_slice(" <", 1).trim_suffix(">");
```

Comment:
```suggestion
					const String website = name.get_slice_count("<") == 1 ? "" : name.get_slice("<", 1).trim_suffix(">");
```
The space is irrelevant here.

Reply:
Just for good measure, I was afraid it would catch false positives but at the same time, Akien wouldn't let that happen I'm sure